paimon-spark/paimon-spark-common/src/test/scala/org/apache/paimon/spark/procedure/ExpireSnapshotsProcedureTest.scala

```diff
@@ -71,7 +71,7 @@ class ExpireSnapshotsProcedureTest extends PaimonSparkTestBase with StreamTest {
 
             // expire
             checkAnswer(
-              spark.sql("CALL paimon.sys.expire_snapshots(table => 'test.T', retain_max => 2)"),
+              spark.sql("CALL paimon.sys.expire_snapshots(table => 'test.T', retain_max => 2, retain_min => 1)"),
```

Comment:
Before, if user not specify the `retain_min`, the default value is 1 in `ExpireSnapshotsImpl`, now the default value is fallback to the `CoreOptions.SNAPSHOT_RETAIN_MIN = 10`, so if max is 2, we should manually specify the `retain_min => 1`. I think the current behavior is more straightforward, I'm not sure whether this will break the compatibility. Please also help check this  cc @JingsongLi